onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1869,7 +1869,8 @@ TEST(InferenceSessionTests, TestLenientShapeInferencing) {
   old_opset.AddInput("data", input_shape, input_data);
   old_opset.AddOutput<int64_t>("output", invalid_output_shape, output_data);
   // TensorRT doesn't handle Unsqueeze
-  old_opset.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
+  // OpenVINO: Disabled temporarily
```

Comment:
more context? is this a regression?

Reply:
Yes they added another optimization pass in OpenVINO which transforms part of the graph and doesn't preserve the output names. I raised a bug for this, but was not fixed by 2021.1. This makes it so that the output names don't match the output names we get from onnx and that's the reason this is failing right now.